<!-- -*- mode: XML -*- -->

<launch>
    <node pkg="ieee2016_xmega_connector" type="xmega_connector_publisher.py" name="xmega_connector" respawn="true" output="screen">
        <rosparam>
            port: /dev/xmega_tty
        </rosparam>
    </node>
</launch>

